sugoi-api-rest-services/src/main/java/fr/insee/sugoi/services/controller/OrganizationController.java

```diff
@@ -309,8 +310,8 @@ public ResponseEntity<?> updateOrganizations(
       Authentication authentication,
       @Parameter(description = "Organization to update", required = false) @RequestBody
           Organization organization) {
-    if (!organization.getIdentifiant().equals(id)) {
-      return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
+    if (!organization.getIdentifiant().equalsIgnoreCase(id)) {
```

Comment:
can you also manage the case when the object doesn't have its id ? (I think it's okay if it is dealt with the same Exception)